eds_scikit/__init__.py

```diff
@@ -9,25 +9,10 @@
     action="ignore", category=FutureWarning
 )  # Remove pyarrow DeprecatedWarning
 
-import importlib
-import os
-import sys
-import time
-from packaging import version
-from typing import List, Tuple
-from pathlib import Path
 
 import pandas as pd
-import pyarrow
-import pyspark
 from loguru import logger
-from pyspark import SparkContext
-from pyspark.sql import SparkSession
-
-import eds_scikit.biology  # noqa: F401 --> To register functions
-
-import eds_scikit.utils.logging
-
+from .io import koalas_options, improve_performances
```

Comment:
Maybe
```suggestion
from eds_scikit.io import koalas_options, improve_performances
```
to be consistent with `eds_scikit.biology` and `import eds_scikit.utils.logging`, but I think both work